### PR TITLE
refactor(pagination): removed av/localstorage for browser localstorage

### LINKED ISSUE #725: Remove @availity/localstorage-core usage
### Current Behavior

@availity/localstorage-core is being used in @availity/pagination, and is a dev or peer dependency in several other packages. 

Examples:

- https://github.com/Availity/availity-react/blob/master/packages/pagination/src/Pagination.js#L63
- https://github.com/Availity/availity-react/blob/master/packages/spaces/package.json#L42

### Expected behavior

@availity/localstorage-core got removed. See https://github.com/Availity/sdk-js/commit/0dfa27385d87e356ce4e2b7a6173d2b1370eb78a . It should not be referenced here anymore

### Suggested solution(s)

Remove dependency usage of local storage core and in the places it's being used, such as @availity/pagination, use the browser's local storage instead.

### PR DESCRIPTION
**Pull request by stranzhay**
Removed @av/localstorage, replaced with browser localstorage fixes #725

### CODE REVIEW

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://616080e61af9dc7d4e67dd95--condescending-leavitt-66e607.netlify.app

File: packages/pagination/package.json [hunk 1]
```diff
@@ -18,7 +18,6 @@
     "url": "https://github.com/Availity/availity-react/issues"
   },
   "dependencies": {
-    "@availity/localstorage-core": "^3.0.0",
```
Comment (jordan-a-young) [inline packages/pagination/package.json:21]: If you removed this manually can you run `yarn install` again so the `yarn.lock` is updated too?

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://61645b94a4b2fce924d1920f--condescending-leavitt-66e607.netlify.app

File: packages/pagination/src/Pagination.js [hunk 2]
```diff
@@ -78,7 +77,7 @@ const Pagination = ({
 
       const pageCount = Math.ceil((totalCount || items.length) / itemsPerPage);
 
-      if (!isEqual(avLocalStorage.get('current-page'), currentPage)) {
+      if (!isEqual(JSON.parse(window.localStorage.getItem('current-page')), currentPage)) {
```
Comment (jordan-a-young) [inline packages/pagination/src/Pagination.js]: Can you pull the `getItem` out into a function that has a `try/catch`? I want to make sure the whole function doesn't blow up if there is an issue with the parse

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://616468e82f501c00d704541b--condescending-leavitt-66e607.netlify.app

**Review by jordan-a-young — APPROVED** [review]

**Pull request merged**